packages/react-composites/src/composites/common/styles/ParticipantContainer.styles.ts

```diff
@@ -94,12 +95,19 @@ export const peoplePaneContainerTokens: IStackTokens = {
 /**
  * @private
  */
-export const participantListWrapper = mergeStyles({
-  width: '20rem',
-  // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
-  maxWidth: '50vw',
-  height: '100%'
-});
+export const participantListWrapper = (theme: Theme): string =>
+  mergeStyles({
+    width: '20rem',
+    // max width at 50% of view so the People Pane is not squeezing the Message Pane to almost nothing when on small screen or high zoom in
+    maxWidth: '50%',
+    height: '100%',
+    position: 'absolute',
```

Comment:
Should we apply these to desktop also? We should look to add another ui test here if so as this covers the sent message here:
![image](https://user-images.githubusercontent.com/2684369/171919410-056306bd-9700-48e6-99f1-b5038045e3c9.png)

Reply:
Maybe UI tests should always hide it by default, and then have a new specific UI test that tests the participant pane